testing/tests/DevExpress.ui.widgets/deferRendering.animation.tests.js

```diff
@@ -23,7 +16,16 @@ QUnit.testStart(function() {
             </div>\
         </div>';
 
+    const style = document.createElement('style');
+    style.innerHTML =
+        '.test-staggering-item,\
+        test-no-staggering-item {\
+            height: 10px;\
+            width: 10px;\
+        }';
+
     $('#qunit-fixture').html(markup);
+    document.body.appendChild(style);
```

Comment:
why is this change necessary?

Reply:
we need to style for the `.test-staggering-item` element, which is in root document, not in shadow root, so we need to attach styles to the body element

Reply:
i think it's good idea to add a comment for such specific case
`// NOTE: staggering item is placed in body outside of shadow root

Reply:
ok